Src/Sarif.PatternMatcher/AnalyzeCommand.cs

```diff
@@ -28,7 +28,7 @@ public AnalyzeCommand(IFileSystem fileSystem = null)
         {
             engine ??= RE2Regex.Instance;
 
-            var validators = new ValidatorsCache();
+            var validators = new ValidatorsCache(fileSystem: fileSystem);
```

Comment:
>fileSystem: fileSystem [](http://example.com/codeflow?start=49&length=22)

Please remove the redundancy in this, i.e., 

pass 
  fileSystem

instead of 
  fileSystem: fileSystem

Make sense? the latter pattern repeats information unnecessarily.

Reply:
This is necessary to disambiguate which nullable parameter we are using to instantiate the ValidatorsCache. Taking this change causes build errors. Chaining the constructors for ValidatorsCache to allow this change, causes build errors elsewhere. This is the cleanest solution.